lib/mappers/digitalnc_mapper.py

```diff
@@ -178,19 +180,25 @@ def map_is_part_of(self):
 
     def map_data_provider(self):
         prop = self.root_key + "note"
-        _dict = {"dataprovider": None}
+        data_providers = []
 
         if exists(self.provider_data, prop):
             for s in iterify(getprop(self.provider_data, prop)):
                 try:
                     if s.get("type") == "ownership":
-                        _dict["dataProvider"] = s.get("#text")
-                        break
+                        data_providers.append(s.get("#text"))
```

Comment:
You can use the `textnode()` method that's in the imports to safely get the text. See elsewhere where it's used.